### ISSUE TITLE
Universal cross-chain sorting

### ISSUE THREAD

**Issue by vikiival** [kodadot/nft-gallery#3138]
I have a feeling that most of the sorting simply does not work

It would be nice to:
- go through the app
- check what does not work
- fix it :)

I think there is a ton of incompatibility issues between the `subquery` and `subsquid` stuff (feel free to help me with `packages/uniquery`)

@roiLeo reported in #3039 that:

-  Sorting in /bsx/collection/{id} is broken

**Comment by petersopko** [comment]
There are only couple things left when it comes to Basilisk, which need to be figured out. I know we've talked about this elsewhere, but regardless:

1. `Recently Interacted` and `Unpopular` are doing the same thing as `Recently Created` and `Oldest`. The reason for this (as far as I can tell) is that activity such as `Offer` isn't changing the `updatedAt` of the NFT, therefore, the NFT get's updated only once and that's when it's created. This filter was created to sort by emotes -> **this is irrelevant on BSX and we should hide it on both /snek and /bsx** (I've checked just now and all of the Basilisk NFTs have `updatedAt` and `createdAt` are equal everywhere, regeradless of activity)
2. `Serial Number Ascending` - in my opinion, this filter is more of a mystery than any added value -> I don't think people know what is this for and I don't think they care for having it. `Oldest` should be enough to sort by `createdAt` -> **let's hide this on both /snek and /bsx**
